src/components/LocationSelector/LocationSelector.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LoadScriptNext } from '@react-google-maps/api';
+import {
+    Modal,
+    Map,
+    FieldWrapper,
+    LocationAutocomplete,
+    Button,
+    LocationCard,
+} from '@textkernel/oneui';
+
+const DEFAULT_LOCATION_CIRCLE_RADIUS = 1;
```

Comment:
I still think this should be a prop, but not required, so that it defaults back to 1.